SOURCES/openvswitch-2.5.3-CVE-2023-5366-odp-ND-Follow-Open-Flow-spec-converting-from-OF-to-DP.backport.patch

```diff
@@ -0,0 +1,187 @@
+From 29b95c356e0e3514d692b5df442ad5a5d94bbee9 Mon Sep 17 00:00:00 2001
+From: David Morel <david.morel@vates.tech>
+Date: Thu, 22 Feb 2024 10:34:52 +0100
+Subject: [PATCH] odp: ND: Follow Open Flow spec converting from OF to DP
+
+From e235a421fbdb0c70176e8a3bef13bf7e2056cbc1 Mon Sep 17 00:00:00 2001
+From: Aaron Conole <aconole@redhat.com>
+Date: Mon, 11 Dec 2023 12:04:19 -0500
+Subject: [PATCH] odp: ND: Follow Open Flow spec converting from OF to DP.
+
+Backport notes:
+- adjusted context as for actual patch, upstream is using helpers now
+  instead of the big if section
+- Added test macros where it seemed to make sense, the other macros do
+  not exist in our version
+- Move actual test at the end fo the datapath section in
+  system-traffic.at as the part it is added to upstream do not exist in
+  our version.
+
```

Comment:
Just a detail, but wouldn't it be better to leave the original commit message untouched and put the backport notes just above, starting line 6?

Reply:
You're right, did that.